cfme/tests/networks/test_provision_to_virtual_network.py

```diff
@@ -0,0 +1,72 @@
+import fauxfactory
+import pytest
+from widgetastic.utils import partial_match
+
+from cfme import test_requirements
+from cfme.common.vm import VM
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.provider([RHEVMProvider],
+                         required_fields=[['provisioning', 'template']],
+                         scope='module'
+                         ),
+    pytest.mark.uncollectif(lambda provider: provider.version < 4.2,
+                            reason='ovn functionality is limited in 4.1'),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.ignore_stream('5.8'),
+    test_requirements.provision,
+]
+
+
+@pytest.fixture(scope='function')
+def network(request, provider, appliance):
```

Comment:
**REQUIRED:** Can we rename either the function, or the variable inside to avoid any confusion.

Reply:
Done.